ipfsconn/ipfshttp/ipfshttp.go

```diff
@@ -806,6 +811,103 @@ func (ipfs *Connector) RepoStat(ctx context.Context) (*api.IPFSRepoStat, error)
 	return &stats, nil
 }
 
+// RepoGC performs a garbage collection sweep on the cluster peer's IPFS repo.
+func (ipfs *Connector) RepoGC(ctx context.Context) (*api.RepoGC, error) {
+	ctx, span := trace.StartSpan(ctx, "ipfsconn/ipfshttp/RepoGC")
+	defer span.End()
+
+	ctx1, cancel1 := context.WithTimeout(ctx, ipfs.config.IPFSRequestTimeout)
```

Comment:
Sorry if it was not clear before, this needs a special timeout, by that I mean a new IPFSGCTimeout option in the config..

Reply:
I tried to check if IPFS keeps any time out when `repo gc` command or API is called, but didn't find anything helpful
have kept this 24 hours, but not sure what should be the right timeout here